spec/koans.js

```diff
@@ -256,18 +257,18 @@ describe('the JavaScript language', () => {
         return internal(4, 4);
       }
 
-      //expect(external()).toBe();
+      expect(external()).toBe(9);
     });
 
-    it('may return arrays that contains closures and so on', () => {
-      function example() {
-        // write the missing code here
-      }
+    // it('may return arrays that contains closures and so on', () => {
```

Comment:
Este punto, que también dejaron comentado, se resolvía de la misma manera que el test `'may return arrays that contains functions and so on'`
```
it('may return arrays that contains closures and so on', () => {
      function example() {
       return [ (arg) => [ 0, 9 + arg  ] ]
      }

      expect(example()[0](1)[1]).toEqual(10);
      expect(example()[0](2)[1]).toEqual(11);
      expect(example()[0](3)[1]).toEqual(12);
    });
```